auxiclean/distributeur.py

```diff
@@ -33,65 +34,16 @@ def _get_cours(self, path):
                 liste.append(c)
         return liste
 
-    def find_best(self, stat_1, stat_2):
-        if stat_1 > stat_2:
-            return True
-        if stat_1 == stat_2:
-            return None
-        else:
-            return False
-
-    def bestmatch(self, nombre_de_poste, liste_de_candidats, nom_du_cours,
-                  liste_des_etudiants):
+    def Filtre(self, nombre_de_poste, liste_de_candidats, liste_des_etudiants):
 
-        classement = []
-        personnes_selectionnees = []
+        np.sort(liste_de_candidats)

```

Comment:
utilise la méthode 'sorted' qui est une fonction built-in de python. Ceci évite d'avoir la dépendance de numpy qui ne sert qu'à ça ici.